apps/charterafrica/src/payload/collections/Tools.js

```diff
@@ -0,0 +1,209 @@
+import {
+  TOOL_COLLECTION,
+  ORGANIZATION_COLLECTION,
+  PEOPLE_COLLECTION,
+} from "../../lib/tools/models";
+import slug from "../fields/slug";
+import source from "../fields/toolSourceField";
+import updatedAt from "../fields/updatedAt";
+
+const Tools = {
+  slug: TOOL_COLLECTION,
+  admin: {
+    useAsTitle: "externalId",
+  },
+  access: {
+    read: () => true,
+  },
+  labels: {
+    singular: {
+      en: "Tool",
+      fr: "Outil",
+      pt: "Ferramenta",
+    },
+    plural: {
+      en: "Tools",
+      fr: "Outils",
+      pt: "Ferramentas",
+    },
+  },
+  fields: [
+    source(),
+    {
+      name: "externalId",
+      type: "text",
+      required: true,
+      label: { en: "External ID", fr: "ID externe", pt: "ID externo" },
+      admin: {
+        readOnly: true,
+      },
+    },
+    slug({ fieldToUse: "name" }),
+    {
+      name: "name",
+      type: "text",
+      label: {
+        en: "Tool Name",
+        fr: "Nom de l'outil",
+        pt: "Nome da ferramenta",
+      },
+      admin: {
+        readOnly: true,
+      },
+    },
+    {
+      name: "description",
+      type: "textarea",
+      admin: {
+        readOnly: true,
+      },
+      label: {
+        en: "Tool Description",
+        fr: "Description de l'outil",
+        pt: "Descrição da ferramenta",
+      },
+    },
+    {
+      name: "link",
+      type: "text",
+      admin: {
+        readOnly: true,
+      },
+      label: {
+        en: "Tool Link",
+        fr: "Lien de l'outil",
+        pt: "Link da ferramenta",
+      },
+    },
+    {
+      name: "location",
+      type: "text",
+      admin: {
+        readOnly: true,
+      },
+      label: {
+        en: "Tool Location",
+        fr: "Emplacement de l'outil",
+        pt: "Localização da ferramenta",
+      },
+    },
+    {
+      name: "topic",
```

Comment:
Are these related to GitHub topics? If yes, a single tool can have multiple topics. See PesaYetu 👇🏽 
<img width="358" alt="Screenshot 2023-05-08 at 09 27 38" src="https://user-images.githubusercontent.com/1779590/236750857-08424c1c-0e39-462e-aa16-969a7e77b4f5.png">

Reply:
Cool @koechkevin . Did you communicate back to the design team that we already have `topics` in GitHub? If they really don't want to use GitHub `topics`, it's best we use a different name for this field in our model to capture what we're getting from the gSheet/Airtable.

Since it's singular, I think `theme`, `category` or `subject` would work best.